yt/frontends/cholla/tests/test_outputs.py

```diff
@@ -0,0 +1,64 @@
+import yt
+from yt.frontends.cholla.api import ChollaDataset
+from yt.testing import assert_equal, requires_file
+from yt.utilities.answer_testing.framework import data_dir_load
+
+_fields = (
+    ("gas", "temperature"),
+    ("gas", "density"),
+)
+
+ChollaSimple = "ChollaSimple/0.h5"
+
+
+@requires_file(ChollaSimple)
+def test_ChollaDataset():
+    assert isinstance(data_dir_load(ChollaSimple), ChollaDataset)
+
+
+@requires_file(ChollaSimple)
```

Comment:
`@requires_file` mark answer tests, but these are simple tests that don't really fit in answer testing.

It would actually be the kind of test we traditionally craft for frontends. See `yt/frontends/amrvac/tests/test_outputs.py` for a collection of simple examples.

Also note that you can't use `yt.load` in an answer test, instead you'll need `yt.utilities.answer_testing.framework.data_dir_load`

Reply:
Thanks for the tips.  I tried using `@requires_ds` instead of `@requires_file` but the testing framework just skipped tests prepended with that, since it wasn't recognizing my dataset as a valid dataset.

Reply:
oh. So sorry I had it the other way around, indeed `requires_ds` is the one that means "answer test", as the very examples I was pointing to in the AMRVAC frontend illustrate 🤦🏻‍♂️ 
Running those locally isn't straightforward though: https://yt-project.org/docs/dev/developing/testing.html#how-to-run-the-answer-tests